src/seedu/addressbook/Main.java

```diff
@@ -84,9 +84,14 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
+
+            try {
+                CommandResult result = executeCommand(command);
+                recordResult(result);
+                ui.showResultToUser(result);
+            }
+            catch (Exception e){
+            }
```

Comment:
What should you do after you catch an exception? Remember what we talked about in tutorial class? :( If you are very confused, please let me know during the next tutorial session!